tests/test_date_parser.py

```diff
@@ -394,106 +491,12 @@ def then_period_is(self, period):
-        date_fixtures = [
-            (u'07/07/2014', datetime(2014, 7, 7)),
-            (u'07.ago.2014 | 12:52', datetime(2014, 8, 7)),
-            (u'07.jul.2014 | 12:52', datetime(2014, 7, 7)),
-            (u'07.feb.2014 | 12:52', datetime(2014, 2, 7)),
-            (u'07.ene.2014 | 12:52', datetime(2014, 1, 7)),
-        ]
-
-        for date_string, correct_date in date_fixtures:
-            self.assertEqual(correct_date.date(), dp.parse(date_string).date())
-
-    def test_should_not_allow_multiple_languages_by_default(self):
-        dates_in_spanish = [
-            (u'13 Ago, 2014', datetime(2014, 8, 13)),
-            (u'11 Marzo, 2014', datetime(2014, 3, 11)),
-            (u'13 Septiembre, 2014', datetime(2014, 9, 13)),
-        ]
-        dp = DateParser()
-
-        for date_string, correct_date in dates_in_spanish:
-            parsed_date = dp.parse(date_string)
-            self.assertEqual(correct_date.date(), parsed_date.date())
-
-        with self.assertRaisesRegexp(ValueError, 'Invalid date'):
-            portuguese_date = u'13 Setembro, 2014'
-            dp.parse(portuguese_date)
-
-    def test_should_enable_redetection_for_multiple_languages(self):
-        dates_fixture = [
-            (u'13 Ago, 2014', datetime(2014, 8, 13)),
-            (u'11 Marzo, 2014', datetime(2014, 3, 11)),
-            (u'13 Septiembre, 2014', datetime(2014, 9, 13)),
-            (u'13 Setembro, 2014', datetime(2014, 9, 13)),
-            (u'13 Março, 2014', datetime(2014, 3, 13)),
-        ]
-        dp = DateParser(allow_redetect_language=True)
-
-        for date_string, correct_date in dates_fixture:
-            parsed_date = dp.parse(date_string)
-            self.assertEqual(correct_date.date(), parsed_date.date())
-
-    def test_finding_no_language_when_not_seen_before_should_raise_error(self):
-        """ This test depends on the defined order of the languages and needs to be rewritten """
-        dp = DateParser()
-        self.assertEqual(datetime(2014, 8, 13).date(),
-                         dp.parse("13 Srpen, 2014").date())
-
-        with self.assertRaises(ValueError):
-            dp.parse('11 Ağustos, 2014')
-
-    def test_finding_no_language_when_enabled_should_redetect(self):
-        dp = DateParser(allow_redetect_language=True)
-        self.assertEqual(datetime(2014, 8, 13).date(),
-                         dp.parse('13 Ago, 2014').date())
-
-        self.assertEqual(datetime(2014, 8, 11).date(),
-                         dp.parse(u'11 Ağustos, 2014').date())
-
-    def test_finding_no_language_should_work_for_numeric_dates(self):
-        dp = DateParser()
-        self.assertEqual(datetime(2014, 8, 13).date(),
-                         dp.parse('13 Ago, 2014').date())
-
-        parsed_date = dp.parse(u'13/08/2014')
-        self.assertEqual(datetime(2014, 8, 13).date(), parsed_date.date())
+    def then_error_was_raised(self, error_cls, error_message):
+        self.assertIsInstance(self.error, error_cls)
```

Comment:
We have this definition at [here](https://github.com/scrapinghub/dateparser/pull/85/files#diff-36128d81467a740315c9e5f49860c291R188) too. Why not move it to `BaseTestCase?`